src/NuGet.Clients/PackageManagement.UI/Converters/IconUrlToImageCacheConverter.cs

```diff
@@ -75,22 +75,27 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
             }
             // if the URL is a file: URI (which actually happened!), we'll get an exception.
             // if the URL is a file: URI which is in an existing directory, but the file doesn't exist, we'll fail silently.
-            catch (Exception e) when (e is System.IO.IOException || e is System.Net.WebException)
+            catch (Exception e)
+            when (
```

Comment:
I thought @rrelyea and I commented here, anyways we want to remove the when, and potentially log to activitylog for any unexpeceted exception here